README.md

```diff
@@ -71,6 +76,19 @@ Fields explanation :
 With PUT *{server_url}/realms/{realmName}/agm/admin/configuration-rules/{id}* you could update a configuration rule.
 With GET *{server_url}/realms/{realmName}/agm/admin/configuration-rules* you could get all configuration rules.
 
+When a group is created, a default configuration is created. Group admin can change it/ create a new one.
+Configuration rules determines the default group configuration and applies rules in group configuration creation/ update.
+
+Default group configuration values without any configuration rules:
+- name →  Join + <group name>
+- Requires approval → True
+- Comments → True
+- Visible → False
+- Active → True
+- No expiration date
+- No aup
+- Default group role (member)
```

Comment:
Could you add the name of the `field` for each config option?
E.g. 
```
- Membership expiration (field `membershipExpirationDays`): ...
```

Reply:
I have created a new table.
Only default group roles is outside it.